ArtemShchepets/src/main/java/week1/controller/ITerminalControllerImpl.java

```diff
@@ -0,0 +1,212 @@
+package week1.controller;
+
+import week1.comparators.SellersSoldProductsComparator;
+import week1.interfaces.IAppDB;
+import week1.interfaces.ITerminalController;
+import week1.model.Bill;
+import week1.model.Product;
+import week1.model.Seller;
+import week1.model.Statistic;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import static week1.utils.TerminalUtils.calculateSumOfSoldProducts;
+import static week1.utils.TerminalUtils.findMaxPriceBill;
+import static week1.utils.TerminalUtils.findMinPriceBill;
+
+public class ITerminalControllerImpl implements ITerminalController {
+
+    private final static Logger logger = Logger.getLogger(ITerminalControllerImpl.class.getName());
+
+    private IAppDB iAppDB;
+
+    public ITerminalControllerImpl(IAppDB iAppDB) {
+        this.iAppDB = iAppDB;
+        logger.setLevel(Level.OFF);
+    }
+
+    @Override
+    public boolean login(String login, String password) {
+
+        iAppDB.setCurrentSeller(-1);
```

Comment:
-1? What is means

Reply:
currentSeller = -1 is a state to show that no one is logged in now. Will refactor this code to more understandable.

Reply:
https://github.com/presly808/aco22/pull/51/commits/b45056f67561831e9fe77997114005ebcd0e7a40 Fixed